docs: clarify what you can build with SvelteKit (#15461)
sveltejs/svelte: documentation/docs/01-introduction/02-getting-started.md

## Alternatives to SvelteKit

> If you're building a [single page app (SPA)](../kit/glossary#SPA), you can [use SvelteKit](../kit/single-page-apps), or you can use Svelte directly with Vite. To create a vanilla Vite project, run `npm create vite@latest` and selecting the `svelte` option. With this, `npm run build` will generate HTML, JS and CSS files inside the `dist` directory using [vite-plugin-svelte](https://github.com/sveltejs/vite-plugin-svelte). In most cases, you will probably need to [choose a routing library](faq#Is-there-a-router) as well.

Reviewer: what about making it a callout note? The current change makes it sound like vanilla Vite is only valid if you're doing SPA.

```suggestion
You can also use Svelte directly with Vite by running `npm create vite@latest` and selecting the `svelte` option. With this, `npm run build` will generate HTML, JS and CSS files inside the `dist` directory using [vite-plugin-svelte](https://github.com/sveltejs/vite-plugin-svelte). In most cases, you will probably need to [choose a routing library](faq#Is-there-a-router) as well.

>[!NOTE] If you only choose vanilla Vite because you're building a [single page app (SPA)](../kit/glossary#SPA), you can also [use SvelteKit](../kit/single-page-apps)
```

Author: >  The current change makes it sound like vanilla Vite is only valid if you're doing SPA.

I think that's basically all vanilla Vite is used for. It's too complex to do SSR without a framework. Was there something else you had in mind that people might build?

Reviewer: I guess that's right, still it feels a bit too technical.
I also noticed just now that the paragraph feels a bit weird because it says "alternatives to SvelteKit" but then starts by stating "use SvelteKit"
